HW0/P5/P5.txt

```diff
@@ -1 +1,17 @@
 # Your answers here.
+1. It takes 255 seconds.
+
+2. Yes, working together makes the verification faster. 
+Each cashier counts the sum of 32 bags using 31 seconds and adding the subtotals up takes 3 seconds, so it takes 34 seconds in total.
+
+3. 2^8 = 256. It takes 8 seconds.
+
+4. See plot. Since the time need to be an integer in seconds, it is [ln(N)/ln(2)]+1, where [] refers to the integer part.
```

Comment:
(Adding my comment here since I can't directly comment on the graph.) To graph the difference between the n and log_2(n), I made the y axis logarithmic. This would make the difference between the two clear for very large n.